djangomodelimport/magic.py

```diff
@@ -115,7 +115,7 @@ def _get_validation_exclusions(self):
         """We need to exclude any CachedChoiceFields from validation, as this
         causes a m * n queries where m is the number of relations, n is rows.
         """
-        exclude = []
+        exclude = super()._get_validation_exclusions()
         for field, fieldinstance in self.fields.items():
             if isinstance(fieldinstance, CachedChoiceField):
                 exclude.append(field)
```

Comment:
`_get_validation_exclusions` returns a set, so should use `.add` here. https://github.com/django/django/blob/main/django/forms/models.py#L390-L433

Reply:
aha, nice catch. I was making assumptions about how this was working beforehand